src/components/ButtonComponents/SpecialButtons/Specials.js

```diff
@@ -1,17 +1,30 @@
 import React from "react";
+import { useState } from "react";
 
 //import any components needed
+import SpecialButton from './SpecialButton.js'
 
 //Import your array data to from the provided data file
+import { specials } from '../../../data.js'
 
 const Specials = () => {
   // STEP 2 - add the imported data to state
+  const [specialState, setSpecialState] = useState(specials);
 
   return (
     <div>
       {/* STEP 3 - Use .map() to iterate over your array data and return a button
        component matching the name on the provided file. Pass
        it any props needed by the child component*/}
+
+      {specialState.map( (item, index) =>
+        <SpecialButton
+          key = {index}
+          test3 = {item}
```

Comment:
Nice work. From a semantic/readability perspective, it might be a good idea to name your props meaningfully (e.g. test3 could be specialChar).